FlightStreamDeck.Logics/ImageLogic.cs

```diff
@@ -77,7 +76,7 @@ public class ImageLogic : IImageLogic
                 {
                     img = defaultBackground;
                 }
-                
+
```

Comment:
I know you are a whitespace stickler 😄

Reply:
It's just Visual Studio auto formatting the file and removing the trailing whitespace 😂.